src/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper/BootstrapperHostingStartup.cs

```diff
@@ -0,0 +1,85 @@
+// <copyright file="BootstrapperHostingStartup.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.AutoInstrumentation.Configuration;
+using OpenTelemetry.Logs;
+
+[assembly: HostingStartup(typeof(OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper.BootstrapperHostingStartup))]
+
+namespace OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper;
+
+/// <summary>
+/// Add summary.
+/// </summary>
+public class BootstrapperHostingStartup : IHostingStartup
+{
+    private readonly LogSettings settings;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="BootstrapperHostingStartup"/> class.
+    /// </summary>
+    public BootstrapperHostingStartup()
+    {
+        settings = LogSettings.FromDefaultSources();
+    }
+
+    /// <summary>
+    /// This method gets called by the runtime to lightup ASP.NET Core OpenTelemetry Logs Collection.
```

Comment:
This is true but the fact that we are currently using it to set up the logs is independent of that. Perhaps separate the log code in a private will make that very clear.